setup.py

```diff
@@ -47,6 +47,8 @@
         'six>=1.10.0',
         'pyyaml>=3.11',
         'wrapt>=1.10.8',
+        'xlrd>=1.1.0',
+        'openpyxl>=2.5.2',
```

Comment:
Why do you need this?

Reply:
`xlrd` is used to import the annotation spreadsheets. `openpyxl` is used to export them.

Reply:
Keep requirements to minimum. Find one stable and well maintained package and use it for all.

Reply:
I guess `openpyxl` is the more important one, because it enables worksheet protection and restricted fields. So, I'll see if it will make sense to rewrite the import part based on that package.

Reply:
I've eliminated the `xlrd` dependency.